server/src/main/java/org/elasticsearch/index/mapper/MappedFieldType.java

```diff
@@ -188,6 +190,10 @@ public boolean isDimension() {
         return false;
     }
 
+    public List<String> dimensions() {
```

Comment:
It's worth adding javadoc to this one just because *everyone* uses `MappedFieldType`.